_build/templates/default/sass/_package-management.scss

```diff
@@ -0,0 +1,69 @@
+#changelog-tab {
+  p {
+    margin-bottom: 0.3rem;
+  }
+
+  h1 {
+    color: #515151;
+  }
+
+  h2 {
+    margin-top: 1rem;
+    font-weight: 700;
+  }
+
+  ul {
+    margin-bottom: 1rem;
+  }
+}
+
+#modx-package-browser-details-main {
+  p {
+    margin-bottom: 0.3rem;
+    color: #808080;
```

Comment:
For normal text, this doesn't pass accessibility, see: https://webaim.org/resources/contrastchecker/?fcolor=808080&bcolor=FFFFFF

To get it to pass would require going to #595959 for the text, and that would also probably need a darker color for the headline for visual balance.